File: src/CadastroUsuarios.Domain/Models/Usuario.cs
using CadastroUsuarios.Domain.Models.Enums;
using System;
using System.Collections.Generic;
using System.Text;

namespace CadastroUsuarios.Domain.Models
{
    public class Usuario
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string Sobrenome { get; set; }
        public string Email { get; set; }
        public DateTime DataNascimento { get; set; }
        public Escolaridade Escolaridade { get; set; }
    }
}

File: src/CadastroUsuarios.FrontEnd/src/app/modules/usuario/model/usuario.ts
import { Escolaridade } from "./enum/escolaridade.model";

export interface Usuario {
  id: number;
  nome: string;
  sobrenome: string;
  email: string;
  dataNascimento: Date;
  escolaridade: Escolaridade;
  escolaridadeString: string;
}
